src/appwindow.cpp

```diff
@@ -1041,7 +1027,10 @@ void AppWindow::onSettingsApplied(const QString &pagePath)
     }
 
     if (pagePath.isEmpty() || pagePath == "Appearance")
+    {
         setWindowOpacity(SettingsHelper::getOpacity() / 100.0);
+        Core::StyleManager::setStyle(SettingsHelper::getUIStyle());
```

Comment:
The reason for your style not being applied correctly the is because it is called way to late, call this inside constructor of Appwindow, where old windows dark theme code was. As I said, in dynamically Changing style, styles will break.

Reply:
I fixed it by setting the palette before the style.

Reply:
Did you succeed in changing styles dynamically?

Reply:
> Did you succeed in changing styles dynamically?

Yes.